src/SingleStoreConnector.DependencyInjection/docs/README.md

```diff
@@ -48,3 +48,33 @@ builder.Services.AddSingleStoreDataSource("Server=server;User ID=test;Password=t
 	x => x.UseRemoteCertificateValidationCallback((sender, certificate, chain, sslPolicyErrors) => { /* custom logic */ })
 );
 ```
+
+## Keyed Services
+
+Use the `AddKeyedSingleStoreDataSource` method to register a `SingleStoreDataSource` as a [keyed service](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#keyed-di-services).
```

Comment:
looks like this link doesn't work, I found a similiar one https://learn.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-9.0#keyed-services not sure if it's referreing to the same mechanism.